.env.example

```
@@ -1,17 +1,20 @@
 # The address where the C++ backend is listening
-AETHERFLOW_GRPC_HOST=localhost
-AETHERFLOW_GRPC_PORT=50051
+AETHERFLOW_GRPC_HOST=
+AETHERFLOW_GRPC_PORT=
 
 # --- Logging & Debugging ---
 # Options: DEBUG, INFO, WARNING, ERROR (matches your .cursorrules)
-LOG_LEVEL=INFO
-DEBUG_MODE=False
+LOG_LEVEL=
+DEBUG_MODE=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DEBUG_MODE key should go before the LOG_LEVEL key

(UnorderedKey)